recipe/meta.yaml

```diff
@@ -61,6 +61,8 @@ test:
   requires:
     - pyyaml
     - mpi * impi  # [not win]
+    - mpi4py
+    - pytest-mpi
```

Comment:
Are we running spmd GPU tests here? Because pytest-mpi is only necessary for the sklearnex.spmd tests on multi GPU